src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/filtering/M2FiltersArgumentCollection.java

```diff
@@ -43,6 +43,13 @@
     @Argument(fullName = M2ArgumentCollection.MITOCHONDRIA_MODE_LONG_NAME, optional = true, doc = "Set filters to mitochondrial defaults")
     public boolean mitochondria = false;
 
+    /**
+     * Mitochondria mode excludes the filters {@link ClusteredEventsFilter}, {@link MultiallelicFilter},
+     * {@link FilteredHaplotypeFilter}, {@link FragmentLengthFilter}, and {@link GermlineFilter}
+     */
+    @Argument(fullName = M2ArgumentCollection.MICROBIAL_MODE_LONG_NAME, optional = true, doc = "Set filters to microbial defaults")
```

Comment:
How difficult would it be to add an integration test to M2 that asserts this argument is doing something reasonable?